Gemfile

```
@@ -3,16 +3,41 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.0'
 gem 'jquery-rails'
-#gem 'bootstrap'
-#gem 'devise_token_auth'
-gem 'rspec', '~> 3.10'
-gem 'annotate'
-#group :development do
-#  gem 'annotate'
-#end
-gem 'rubocop', '~> 1.15'
-gem 'reek', '~> 6.0', '>= 6.0.4'
-gem 'rails_best_practices', '~> 1.20', '>= 1.20.1'
+gem 'rspec-rails'
```

Please add the gem version, same for the other gems
```suggestion
gem 'rspec-rails', '~> 5.0', '>= 5.0.1'
```

---

Also, this gem should be only available for development and test environment